7.1 Snapshot.md

```diff
@@ -0,0 +1,52 @@
+
+Transactions are a special type of message, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is omitted in this first iteration of the protocol.
+
+## 7.1.5 Pruning dependencies
+
+We require several conditions to be fulfilled in order to allow for a smooth pruning process. Pruning is done when messages reach the age `DELTA_PRUNE`. 
+
+Since messages cannot attach to a message whose timestamp has an age difference of more than `DELTA`, see [4.3.4 R-URTS](./4.3%20Tip%20Selection%20Algorithm.md/#4.3.4%20R-URTS), and the timestamp precision is enforced through voting, see [4.2.3 General Timestamp rules](./4.2%20Timestamps.md/#4.2.3%20General%20Timestamp%20rules). Finalization of messages must be guaranteed to happen before the point of pruning. Thus, any message must be either confirmed or rejected before the elapse of the pruning age. This requires that both voting on timestamps and conflicts, as well as the accumulation of sufficient approval weight takes a time much smaller than `DELTA_PRUNE`. 
+
+In order to allow a smooth pruning, any message at the age `DELTA_PRUNE` must either belong to the branch `masterBranch` or `rejectedBranch`. This simply means that messages are either confirmed or rejected. 
+
+Transaction must pass a past cone check, see [### 2.4.4.6 Booker - Point 7.](./2.4%20Data%20Flow.md/#2.4.4.6%20Booker). However the past cone check cannot apply to messages below a certain age `DELTA_PASTCHECK<<DELTA_PRUNE`. Thus, the past cone check there needs to be relaxed (???TODO).
+
+With the above conditions we can safely assume any message before the time `DELTA_PRUNE>>DELTA` can be pruned. 
+
+## 7.1.6 Pruning process
+
+An output is considered *confirmed* spent, when the transaction spending an output is confirmed. A flag `confirmedSpent` *shall* be set to `TRUE` in the metadata of the output when this event is triggered.
+
+Once a message reaches the age `DELTA_PRUNE` several checks are performed:
+
++ If the message is a data message it can be removed. 
++ If the message is a value message and is rejected, i.e. is member of the `rejectedBranch` it can be removed.
++ If the message is a value message and is confirmed, i.e. is member of the `masterBranch` it is checked, whether any of the outputs is unspent, i.e. not confirmed spent. 
++ If all outputs are confirmed spent the value message is removed. 
```

Comment:
This is conflicting with "Transactions are a special type of message, that cannot be pruned.", we need to change this.